src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,20 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import React from 'react';
+
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo:React.FC<Props> = ({ todo }) => (
+  <article className={`TodoInfo ${todo.completed && 'TodoInfo--completed'}`}>
+    <h2 className="TodoInfo__title">
+      {todo.title}
+    </h2>
+
+    {todo.user && (
+      <UserInfo user={todo.user} />
+    )}
+  </article>
 );
```

Comment:
Use destructuring assignment.

When you check `completed` in className and get false it will return false to classname. 
Tyr to use ternary operator here

![image](https://github.com/mate-academy/react_static-list-of-todos/assets/84296041/6de88809-8ebc-4d05-a01e-dabeacb4336e)


```suggestion
export const TodoInfo:React.FC<Props> = ({ todo }) => {
  const {
    completed,
    title,
    user
  } = todo;
  
  return (
    <article className={`TodoInfo ${completed ? 'TodoInfo--completed' : ''}`}>
      <h2 className="TodoInfo__title">
        {title}
      </h2>

      {user && (
        <UserInfo user={user} />
      )}
    </article>)
};
```